### PR TITLE
localize skeleton accent

### PR DESCRIPTION
**Pull request by lzk228**
just put strings out of system for localizing

### CODE REVIEW

File: Resources/Locale/en-US/accent/skeleton.ftl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+accent-skeleton-words-1 = fuck you
+accent-skeleton-words-replace-1 = I've got a BONE to pick with you
+
+accent-skeleton-words-2 = fucked
+accent-skeleton-words-replace-2 = boned
+
+accent-skeleton-words-3 = fuck
+accent-skeleton-words-3-2 = fck
+accent-skeleton-words-3-3 = shit
+accent-skeleton-words-replace-3 = babushka
```
Comment (slarticodefast) [inline Resources/Locale/en-US/accent/skeleton.ftl:10]: wtf?

File: Resources/Locale/en-US/accent/skeleton.ftl [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+accent-skeleton-words-4 = definitely
+accent-skeleton-words-4-2 = definitely
```
Comment (slarticodefast) [inline Resources/Locale/en-US/accent/skeleton.ftl:13]:
```suggestion
accent-skeleton-words-4-2 = absolutely
```

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/accent/skeleton.ftl (hunk 1 above)
Comment (lzk228) [reply Resources/Locale/en-US/accent/skeleton.ftl:10]: ahhyes i was copying "preset" from russian and then replaced words

**Comment by slarticodefast** [comment]
Looks good, thanks for the fix!

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**